cmd/stat.go

```diff
@@ -109,12 +109,27 @@ func (stat statMessage) String() (msg string) {
 	}
 
 	if maxKeyEncrypted > 0 {
-		if keyID, ok := stat.Metadata["X-Amz-Server-Side-Encryption-Aws-Kms-Key-Id"]; ok {
+		// Handle various AWS S3 headers, behaviors etc.
+		var found bool
+		if enabled, ok := stat.Metadata["X-Amz-Server-Side-Encryption-Bucket-Key-Enabled"]; ok {
+			if enabled == "true" {
+				msgBuilder.WriteString(fmt.Sprintf("%-10s: SSE-%s\n", "Encryption", "KMS"))
+			}
+			found = true
```

Comment:
IIUC, `found` is true when we find a supported encryption mechanism used. When `enabled` is false, it is as good as unknown to MinIO, correct?
```suggestion
				msgBuilder.WriteString(fmt.Sprintf("%-10s: SSE-%s\n", "Encryption", "KMS"))
				found = true
			}
```

Reply:
no found is true when `enabled = false` as well @krisis it means we don't need to say "unknown" its a special case where we need to treat it as SSE not set.